buyTogetherApp/templates/buyTogetherApp/main.html

```diff
@@ -0,0 +1,131 @@
+                text-align: left;
+                padding: 8px;
+              }
+              
+              tr:nth-child(even) {
+                background-color: #dddddd;
+              }
+              .projectName{
+                  text-align: center;
+                  font-size: 100px;
+                  color: purple;
+              }
+              .bigFont{
+                  font-size: 30px;
+                  margin-top: 80px;
+                  margin-left: 20px;
+                  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+              }
+              .red{
+                  color: red;
+              }
+            .blue{
+                color: blue;
+            }
+
+        </style>
+    </head>
+
+    <body>
+        <div class="projectName red">
+            buyTogether
+        </div>
+        <div class="bigFont">
+            <p>The idea of the project is to let group of people to organize together and buy straight from the supplier at lower cost.</p>
+            Goals
+            <br>
+            - to create simple and easy communication between an organized group of people and a supplier.
+            <br>
+            - lower the prices of food cost
+            <br>
+            Our plan is to create a website that will communicate between the group of people and the supplier, removing the supermarkets from the equation, which will lead to lower the price for the customer.
+        </div>
+        <br>
+        <table>
+            <tr>
+              <th>Name</th>
+              <th>Year</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td>Elipaz Menachem</td>
+              <td>28 years old</td>
+              <td>third year of computer science degree</td>
+            </tr>
+            <tr>
+              <td>Lior Silberman</td>
+              <td>27 years old</td>
+              <td>I'm a third year computer science student</td>
+            </tr>
+            <tr>
+              <td>Meitar Rizner</td>
+              <td>26 years old</td>
+              <td>A third year computer science student</td>
+            </tr>
+            <tr>
+              <td>Rawan Abuleil</td>
+              <td>23 years old</td>
+              <td>third year of computer science degree</td>
+            </tr>
+            <tr>
+              <td>Karam Jbareen</td>
+              <td>23 years old</td>
+              <td>fourth year of computer science degree</td>
+            </tr>
+          </table>
+          <p class="bigFont">
+              We will all be working and helping each other in each field and each division but basically ,2 will be working on the front end and 2 on the back end and one on the communication between the group and QR checking codes and all together will be managing the database and models.
+          </p>
+          <p class="bigFont">
+           ***** Contributing will be more detailed on the Contributing.md *****<br>
```

Comment:
By adding this stuff here, you're introducing duplicate code. It means that if change your `CONTRIBUTING.md`, you will also have to change this content, which essentially couples the PRs for those two files.

Your options:

1. Have those two PRs and change them together (doesn't smell right to me)
2. Have a dynamic link here to the `CONTRIBUTING.md`, which will at least reduce the duplicated code. This still makes those PRs dependent on each other which makes things more complex, and it also introduce a technical challenge which is how to make this dynamic link between this page and the file.
3. Drop this content from this PR and find another way to present the contributing guide other than this web page.

Reply:
Done